plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSourceProvider.java

```diff
@@ -210,7 +211,8 @@ public ConnectorPageSource createPageSource(
                 parquetDateTimeZone,
                 fileFormatDataSourceStats,
                 options,
```

Comment:
Since we are re-using the ParquetPageSourceFactory from hive here and ParquetReaderOptions has bloom filters enabled by default, I think bloom filters will start getting used by default in Delta Lake as well.
You can prevent by adding `.withBloomFilter(false)` during construction of `options` here and in other relevant places in delta lake.

Reply:
thanks, working on it.

Reply:
hi, I added config for deltaLake, for Iceberg and Hudi, since it directly calls predicateMatches() and pass in Optional.empty() as bloom filter store, bloom filter is also disabled.

Reply:
Hi, thanks for bring up this issue, I changed the default value for `ParquetReaderOptions` has bloom filters as disabled. To make sure deltaLake by default has bloom filter disabled(since we haven't test it) 

We can turn this default to enabled once we tested it on iceberg, Hudi and delta lake.

At the meantime, user can use this feature by setting session properties:
`"set session hive.parquet_use_bloom_filter=true"
`

How do you think?

Otherwise, we need to explicitly disable it in delta lake, whereas  
1. `parquetReaderConfig.toParquetReaderOptions()` were called
2.  `new ParquetReaderOptions()` were called

Reply:
Short summary:
1. `DeltaLakeMergeSink` from `new ParquetReaderOptions()`
2. `DeltaLakePageSourceProvider`: from `parquetReaderConfig.toParquetReaderOptions()`, then one place added extra config and another place didn't
3. `DeltaLakeUpdatablePageSource` created at runtime
4. `TransactionLogAccess`: from `parquetReaderConfig.toParquetReaderOptions()`
5. `TableSnapShot`: from `TransactionLogAccess`
6. `CheckpointEntryIterator`: from `TableSnapShot`

Summary:
`CheckpointEntryIterator`
	`ParquetPageSourceFactory.createPageSource()`(at constructor): 
		from `TableSnapShot` when initialize `CheckpointEntryIterator`

`TableSnapShot`
	`new CheckpointEntryIterator`(at `getCheckpointTransactionLogEntries()`):
		from constructor when initialize `TableSnapShot`
	new `TableSnapshot`: 
		(at` getUpdatedSnapshot()`): from constructor when initialize `TableSnapShot`
		(at static` load()`): from function parameters -> from `TransactionLogAccess`
	constructor:
		from static` load()`

`TransactionLogAccess`
	`TableSnapshot.load()`(at `loadSnapshot()`): 
		from` parquetReaderConfig.toParquetReaderOptions()`(when initialize `TransactionLogAccess`)

`DeltaLakeMergeSink`
 	`ParquetPageSourceFactory.createPageSource()`:
		from new `ParquetReaderOptions()`

`DeltaLakePageSourceProvider`
	new `DeltaLakeUpdatablePageSource`: 
		initialized at `parquetReaderConfig.toParquetReaderOptions()`
	`ParquetPageSourceFactory.createPageSource`: 
		initialized at `parquetReaderConfig.toParquetReaderOptions()` then added extra config in `createPageSource()`

`DeltaLakeUpdatablePageSource`: 
	`ParquetPageSourceFactory.createPageSource`
		created at runtime: `parquetReaderOptions.with...`